testing/dataflowexample/dataflow-setup.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
move this setup to CFI's setup script, other solvers use lingeling and everyone can use dljc